.storybook/addons/storybook-addon-html/register.js

```diff
@@ -0,0 +1,29 @@
+// copied from original NPM package https://github.com/whitespace-se/storybook-addon-html
```

Comment:
Copying vendor code and keeping as vendor code is not a good idea. Either use NPM directly as depency and if not possible like here, write the code like our own. Since we dont code in ES5, please change everything accordingly to eslint and ES6+